test/system/flows/accept_mentor_request_test.rb

```diff
@@ -78,6 +78,114 @@ class AcceptMentorRequestTest < ApplicationSystemTestCase
       end
     end
 
+    test "mentor sees locked until information before sending first message" do
+      mentor = create :user, handle: "author"
+      student = create :user, handle: "student"
+      solution = create :concept_solution, user: student
+      request = create :mentor_request, solution:, comment_markdown: "How to do this?",
+        updated_at: 2.days.ago
```

Comment:
Try and ensure that all the fields that you specify are needed/used. So if we don't need this specific markdown or this specific `updated_at` then don't specify them when creating from factories.